database/helper.js

```diff
@@ -1,22 +1,45 @@
-const model = require('../database/models');
+const { client } = require('./client');
 
 module.exports.addBookingDetail = (obj) => {
   const {
-    listingId, searchId, neighbourhood, roomType, nightlyPrices,
+    listingId, searchId, neighbourhood, roomType, nightlyPrices, market, averageRating,
   } = obj;
-  const bookingObj = {
-    listing_id: listingId, market: 'San Francisco', neighborhood: neighbourhood, room_type: roomType, review_scores_rating: Math.floor(Math.random() * 100),
-  };
-  const nightlyEntries = nightlyPrices.map((nightlyPrice) => {
-    const nightlyObj = {
-      listing_id: listingId,
-      date: nightlyPrice.date,
-      price: nightlyPrice.price,
-      search_id: searchId,
-    };
-    return model.NightlyPrices.forge(nightlyObj).save();
+  const bookingText = `INSERT INTO listings (listing_id, market, neighbourhood, room_type, review_scores_rating) 
+    VALUES (${listingId}, '${market}', '${neighbourhood}', '${roomType}', ${averageRating}) 
+    ON CONFLICT (listing_id) DO NOTHING`;
+  return client.query(bookingText)
+    .then(() => {
+      const nightlyEntries = nightlyPrices.map((night) => {
+        const nightlyText = `INSERT INTO booked_nights (listing_id, booked_at, price, search_id) 
+          VALUES (${listingId},'${night.date}'::date,${night.price},${searchId}) `;
```

Comment:
Inconsistent spacing -- the spacing used in the other lines read more easily.

Reply:
Fixed spacing: VALUES (${listingId}, '${night.date}'::date, ${night.price}, ${searchId})`;